interactive_templates/templates/v2/tests/test_event_counts.py

```diff
@@ -0,0 +1,39 @@
+import pandas as pd
+from analysis.event_counts import get_summary_stats, round_to_nearest
+from hypothesis import given
+from hypothesis import strategies as st
+
+
+@given(
+    st.integers(max_value=100_000_000, min_value=0), st.integers(min_value=1)
+)  # Generate random integers for x and base (base > 0)
+# Possible values for x are clamped. For large numbers the test will fail due to the limited precision of
+# floating-point arithmetic
+def test_round_to_nearest(x, base):
+    result = round_to_nearest(x, base)
+
+    # Check if the result is a multiple of base
+    assert result % base == 0
+
+    # Check if the difference between result and x is greater than or equal to -base / 2
+    # This ensures the result is rounded to the nearest value of base and not further away
+    assert result - x >= -base / 2
+
+    # Check if the difference between result and x is less than or equal to base / 2
+    # This also ensures the result is rounded to the nearest value of base
+    assert result - x <= base / 2
+
+
+def test_get_summary_stats():
+    data = {
+        "patient_id": [1, 1, 2, 3, 4],
+        "event_measure": [1, 0, 1, 1, 0],
+        "practice": ["A", "A", "B", "C", "C"],
+    }
+    df = pd.DataFrame(data)
+    summary_stats = get_summary_stats(df)
+
+    assert len(summary_stats["unique_patients"]) == 4
+    assert summary_stats["num_events"] == 3
+    assert len(summary_stats["unique_practices"]) == 3
+    assert len(summary_stats["patients_with_events"]) == 3
```

Comment:
These don't seem especially high-value. Have I missed something? Is the length of each list its most important property?

Reply:
You're right, this is a bit lazy. I'll update them to make some assertions about the contents